src/test/java/org/takes/rq/RqFormTest.java

```diff
@@ -86,4 +86,49 @@ public void sameInstance() throws IOException {
             Matchers.is(Boolean.TRUE)
         );
     }
+
+    /**
+     * RqForm.Fake can create fake forms with parameters list.
+     * @throws IOException If something goes wrong.
+     */
+    @Test
+    public void createsFormRequestWithParams() throws IOException {
+        final String key = "key";
+        final String akey = "anotherkey";
+        final String value = "value";
+        final String avalue = "a&b";
+        final String aavalue = "againanothervalue";
+        final RqForm req = new RqForm.Fake(
+            new RqFake(),
+            key, value,
+            key, avalue,
+            akey, aavalue
+        );
+        MatcherAssert.assertThat(
+            req.param(key),
+            Matchers.hasItems(value, avalue)
+        );
+        MatcherAssert.assertThat(
+            req.param(akey),
+            Matchers.hasItems(aavalue)
+        );
+        MatcherAssert.assertThat(
+            req.names(),
+            Matchers.hasItems(key, akey)
+        );
+    }
+
+    /**
+     * RqForm.Fake throws an IllegalArgumentException when invoked with
+     * wrong number of parameters.
+     * @throws IOException If something goes wrong.
+     */
+    @Test(expected = IllegalArgumentException.class)
+    public void throwsExceptionWhenNotCorrectlyCreated()
+        throws IOException {
```

Comment:
@erimerturk Here too, it should declare `throws Exception`, as per the recommendation [in this post](http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html?expand=1#test-method-names).